modules/cluster-ingress/resources/gateway.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: eg
+  annotations:
+    cert-manager.io/cluster-issuer: <cluster-issuer-placeholder>
```

Comment:
Is this where we have to add the value back in after the resource is deployed?

Reply:
This is handled in the `modules/cluster-ingress/main.tf` file:

```
    kustomize:
      patches:
      - target:
          kind: Gateway
        patch: |-
          - op: replace
            path: /metadata/annotations/cert-manager.io~1cluster-issuer
            value: ${var.cluster_issuer_name}
```